packages/admin-stories/src/docs/components/Form/Form.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/Form" />
+
+# Form
+
+tba.
+
+-   FinalForm
+-   Field
+-   present custom fields
+-   Styling
+-   submit mutation best practice
+
+## Validation
+
+Form validation can be done in two ways, either for the whole form (record-level), or for a single filed (field-level).
+
+> For more information regarding validation, please review the React Final Form documentation on [record-level validation](https://final-form.org/docs/react-final-form/types/FormProps#validate) and [field-level validation](https://final-form.org/docs/react-final-form/types/FieldProps#validate).
+
+### Record-level validation
+
+<Canvas>
+    <Story id="stories-components-form--record-level-validation" />
+</Canvas>
+
+> Use the special key `FORM_ERROR` to display a validation error concerning the whole form.
+
+### Field-level validation
+
+<Canvas>
+    <Story id="stories-components-form--field-level-validation" />
+</Canvas>
+
+### Warnings
+
+In addtion to error validation, @comet/admin also supports warnings for the `FinalForm` and `Field` components. This can be done by using the `validateWarning` props, which offers the same functionality as the React Final Form `validate` props.
```

Comment:
```suggestion
In addtion to error validation, @comet/admin also supports warnings for the `FinalForm` and `Field` components. This can be done by using the `validateWarning` props, which offer the same functionality as the React Final Form `validate` props.
```